modules/update-service-overview.adoc

```diff
@@ -9,53 +9,28 @@
 // * updating/updating-disconnected-cluster.adoc
 
 [id="update-service-overview_{context}"]
-= About the {product-title} update service
+= About the OpenShift Update Service
 
-The {product-title} update service is the hosted service that provides over-the-air
-updates to both {product-title} and {op-system-first}. It provides a graph,
-or diagram that contain _vertices_ and the _edges_ that connect them, of
-component Operators. The edges in the graph show which versions you can safely
-update to, and the vertices are update payloads that specify the intended state
-of the managed cluster components.
+The OpenShift Update Service (OSUS) is the service that provides over-the-air updates to both {product-title} and {op-system-first}. It provides a graph, or diagram, that contains _vertices_ and the _edges_ that connect them, of component Operators. The edges in the graph show which versions you can safely update to, and the vertices are update payloads that specify the intended state of the managed cluster components.
```

Comment:
Looks like we only refer to the initialism in this one spot, so I'm removing `OSUS` because elsewhere in the PR it's always spelled out.

Reply:
We mention OSUS in other places like in the container image metadata etc. It would be a good idea to say that OSUS stands for OpenShift update service.

Reply:
Ah, good point. I've added it back in, thanks.